File: 4st/Hani/programmer/모음제거.js
function solution(my_string) {
  var answer = "";

  for (let ch of my_string) {
    if (ch != "a" && ch != "e" && ch != "i" && ch != "o" && ch != "u") {
      answer += ch;
    }
  }

  return answer;
}

File: 5st/gyuri/모음 제거.java
class Solution {
    public String solution(String my_string) {
        String answer = "";
        
        for(int i=0; i<my_string.length(); i++){
            char c = my_string.charAt(i);
            if(c != 'a' && c != 'e' && c != 'i' && c != 'o' && c != 'u'){
                answer += c;
            }
        }
        return answer;
    }
}